packages/react/src/components/Footer/LanguageSelector.js

```diff
@@ -5,11 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, { useState, useEffect, useRef } from 'react';
-import ComboBox from '../../internal/vendor/carbon-components-react/components/ComboBox/ComboBox';
+import React, { useState } from 'react';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
 import PropTypes from 'prop-types';
-import root from 'window-or-global';
+import Select from '../../internal/vendor/carbon-components-react/components/Select/Select';
+import SelectItem from '../../internal/vendor/carbon-components-react/components/SelectItem/SelectItem';
```

Comment:
I thought the idea was that we would only use select for mobile/tablet, and the combobox remains for desktop?

Reply:
@jeffchew Yep, you are right. Confirmed with @wonilsuhibm that it should use combobox for desktop.